paddle/fluid/framework/paddle2cinn/build_cinn_pass.cc

```diff
@@ -545,6 +545,15 @@ void ReplaceSubGraphWithCinnOpNode(
   RemoveSubGraphFromGraph(cluster, cluster_internals, graph);
 }
 
+static bool IsInplaceOp(const OpDesc& op_desc) {
```

Comment:
Inplace op is not supported in CINN because of the following codes.

https://github.com/PaddlePaddle/CINN/blob/d7a2c0187e59c1b98b17eece9c8e00ce42e767d9/cinn/frontend/op_mapper_registry.cc#L26